src/components/Auth/AuthConfig.tsx

```diff
@@ -0,0 +1,179 @@
+import { loader } from './assets/loader'
+import { success } from './assets/success'
+import { borderRadius, color } from '@onfido/castor'
+import { FaceTecCustomization } from '~auth-sdk/FaceTecSDK.js/FaceTecCustomization'
+import { FaceTecSDK } from '~auth-sdk/FaceTecSDK.js/FaceTecSDK'
+import { UICustomizationOptions } from '../../types/ui-customisation-options'
+
+const ConfigBuilder = () => {
+  const BaseURL = process.env.AUTH_URL
+  return {
+    BaseURL,
+    getAuthCustomization,
+  }
+}
```

Comment:
Sorry for not spotting this earlier, but IMO simple exported const could serve you well here, instead of wrapping them in an additional object like this.